CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Tue Dec 04 2018 Jeanne Greulich <jeanne.greulich@onyxpoint.com> - 6.6.0-0
+- Patch to add space when joining back the elements of command.
```

Comment:
Would you please add more details to this (e.g. which command, which elements, manifest affected)